admin/kernel-masterdata-service/src/main/java/io/mosip/kernel/masterdata/service/impl/GenericServiceImpl.java

```diff
@@ -48,4 +50,22 @@
 		}
 		return list;
 	}
+
+	public List<DocMissingData> getMissingsDetails(Class entity, String langCode, String fieldName) {
+
+		List<MissingDataDto> lst = getMissingData(entity, langCode, "name", fieldName);
+		List<DocMissingData> doc = new ArrayList<>();
+		for (int i = 0; i < lst.size(); i++) {
+
+			DocMissingData d = new DocMissingData(
+					null == lst.get(i).getFieldValue() ? lst.get(i).getId() : lst.get(i).getFieldValue(),
```

Comment:
Why are you setting value to Id and Id to value ?

Reply:
In common query for fetching missing details, if 'code' is passed as idfieldname query is not fetching any data because where clause of query will be built as 'code is null', hence 'name' is passed as idfieldname wherein data which is fetched from common logic will be as follows
{
    id : 'Un justificatif de domicile',
    fieldvalue : 'POA'
    langcode : 'fra'
}		
but actually needed  as 
{
    id: 'POA',
    fieldvalue : 'Un justificatif de domicile'
    langcode : 'fra'
}		
As per discussion with Dhanendra had to swap values of id and fieldvalue